### DISCUSSION TITLE
Getting a console warning

### DISCUSSION THREAD

**Discussion by fsteenkamp** [ueberdosis/tiptap#1347] [General Questions] [answered]
Hi there,
I'm getting a console log as follows on my Editor:
```
[tiptap warn]: "@tiptap/extension-collaboration" comes with its own history support and is not compatible with "@tiptap/extension-history".
```

This is the editor definition:
```
const editor = useEditor({
    extensions: [
      StarterKit,
      Placeholder,
      Collaboration.configure({
        document: ydoc,
      }),
      CollaborationCursor.configure({
        provider: provider,
        user: {
          name: user.email,
          color: getRandomColor(),
        },
      }),
    ],
  });
```

Everything is working perfectly, so no bugs or problems. I have not implemented or even downloaded the history extension. This log isn't a concern, of course, just wondering if I'm doing something wrong. 😄

**Comment by fsteenkamp** [comment]
Is the history extension maybe part of the StarterKit? Not too sure what's included in there 🤔

**Comment by fsteenkamp** [comment]
Ok the history extension was in the starter kit. My ugly solution is this:

```
const starterKit = StarterKit;
starterKit.options.history = false;
```

Any better solutions are welcome 😅

**Reply by hanspagel** [reply]
Yes! StarterKit({ history: false })

**Reply by philippkuehn** [reply]
nope. `StarterKit.configure({ history: false })`

**Comment by fsteenkamp** [comment, marked as answer]
Answer: `StarterKit.configure({ history: false })`
